w3f/substrate-benchmarks/roles/gcp/vars/main.yml

```diff
@@ -1,11 +1,11 @@
 ---
 # vars file for gcp
-desired_state: "{{ state | default('absent') }}"
-gcp_project:  "{{ set_gcp_project | default('polkadot-benchmarks') }}"
+gcp_project:  "{{ lookup('env', 'GCP_PROJECT') | default('polkadot-benchmarks') }}"
+gcp_cred_file: "{{ lookup('env', 'GOOGLE_APPLICATION_CREDENTIALS_CONTENT') }}"
```

Comment:
`gcp_cred_file` should contain the path to the credentials file and the `GOOGLE_APPLICATION_CREDENTIALS_CONTENT` env var has the contents of the credentials file. You should previously store this contents on a file with the path stored in `GOOGLE_APPLICATION_CREDENTIALS` and set that variable instead here.